src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -457,18 +457,24 @@ void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp, const MmapPerfEven
   ORBIT_CHECK(listener_ != nullptr);
   ORBIT_CHECK(current_maps_ != nullptr);
 
-  // Obviously the uprobes map cannot be successfully processed by orbit_object_utils::CreateModule,
-  // but it's important that current_maps_ contain it.
-  // For example, UprobesReturnAddressManager::PatchCallchain needs it to check whether a program
-  // counter is inside the uprobes map, and UprobesUnwindingVisitor::Visit( const
-  // StackSamplePerfEvent*) needs it to throw away incorrectly-unwound samples.
-  // As below we are only adding maps successfully parsed with orbit_object_utils::CreateModule, we
-  // add the uprobes map manually. We are using the same values that that uprobes map would get if
-  // unwindstack::BufferMaps was built by passing the full content of /proc/<pid>/maps to its
-  // constructor.
-  if (event_data.filename == "[uprobes]") {
```

Comment:
So this is not needed anymore, as we cover this with the cases below, right?

Reply:
Maybe add a short comment stating that uprobes are covered.

Reply:
Exactly. I made sure to manually test this case specifically. I carried over some of the old comment and adjusted.